### ISSUE TITLE
No normal map support and too much transparency

### ISSUE THREAD

**Issue by Wunder-Wulfe** [Grim-es/material-combiner-addon#15]
While this tool *does* save a lot of time, it is missing a lot of features. One of the most important would be to save atlased versions of the normal maps and emissions maps etc as well. It also improperly maps textures and leaves huge open transparent spaces that will make the image have a much larger resolution and it will be downscaled in the engine.

**Comment by Wunder-Wulfe** [comment]
Here's a perfect example with the transparency issue:
https://puu.sh/EXoMI/b8b59e3474.png

**Comment by Grim-es** [comment]
Hi, the feature to combine several materials (such as normal maps/etc) currently only available on 2.0.3.3 version of the addon for blender 2.79. The bin packing algorithm is not perfect and leaves a lot of empty space, but to reduce this amount you can combine through MatCombiner panel with Atlas size: Automatic and gaps between images: 0, (combining through Cats addon uses Power of 2 size and gaps 16). To convert and work with images uses the Pillow image library, so the transparency issue may be related to this library, you can try converting textures to png before combining.

**Comment by Wunder-Wulfe** [comment]
Ok, thanks. My last suggestion would be (to at least) add an option to crop the images and scale the UV accordingly before or after finalizing the image. I can cut out the transparency myself, however I would also need to rescale the UV which is difficult to align manually at times.

**Comment by Kosyne** [comment]
having to do normal maps by hand after combining with this plugin's been a major pain point for sure. Has there been any plans to revisit this functionality in the future?

**Comment by whattheheckman** [comment]
Yeah, the lack of PBR support is killing me right now. This addon would be a great time saver but since PBR is a must for my project, I'm finding it easier to create the atlases from scratch then trying to match what the addon is doing. This way can have control over where my UV's are going and an easier time using different shaders for the mesh. If the addon still supported PBR I wouldn't have to do this but unfortunately I (and I'm sure many others) have to atlas manually now. Kinda defeats the point of the addon IMO.

If I may ask @Grim-es, what needs to be done to bring multi combining support back to this addon?
